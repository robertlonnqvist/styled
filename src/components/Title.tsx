import React from "react";

interface TitleProps {
  title: string;
}

const Title = ({ title }: TitleProps): JSX.Element => {
  return (
    <div className="flex justify-between items-center px-4 py-2 bg-black rounded-t-lg">
      <div className="flex justify-start space-x-2">
        <div className="w-3 h-3 bg-red-500 rounded-full"></div>
        <div className="w-3 h-3 bg-yellow-400 rounded-full"></div>
        <div className="w-3 h-3 bg-green-400 rounded-full"></div>
      </div>
      <div className="text-center flex-grow">{title}</div>
    </div>
  );
};

export default Title;
