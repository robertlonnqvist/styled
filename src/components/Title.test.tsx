import React from "react";
import { render } from "@testing-library/react";
import Title from "./Title";

test("renders Nav component", () => {
  const { container } = render(<Title title="test" />);
  expect(container).toMatchSnapshot();
});
