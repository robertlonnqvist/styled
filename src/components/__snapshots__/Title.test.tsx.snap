// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Nav component 1`] = `
<div>
  <div
    class="flex justify-between items-center px-4 py-2 bg-black rounded-t-lg"
  >
    <div
      class="flex justify-start space-x-2"
    >
      <div
        class="w-3 h-3 bg-red-500 rounded-full"
      />
      <div
        class="w-3 h-3 bg-yellow-400 rounded-full"
      />
      <div
        class="w-3 h-3 bg-green-400 rounded-full"
      />
    </div>
    <div
      class="text-center flex-grow"
    >
      test
    </div>
  </div>
</div>
`;
